examples/python/README.md

```diff
@@ -0,0 +1,170 @@
+# Connecting to AlloyDB
+
+## Before you begin
+
+## Before you begin
+
+1. Enable access to AlloyDB in your project by following these [instructions](https://cloud.google.com/alloydb/docs/project-enable-access)
+
+1. Create a VPC network and [configure Private Services Access for AlloyDB](https://cloud.google.com/alloydb/docs/configure-connectivity)
+
+1. Create an AlloyDB cluster and its primary instance by following these [instructions](https://cloud.google.com/alloydb/docs/cluster-create). Make note of the Cluster ID, Instance ID, IP Address and Password
+
+1. Create a database for your application by following these 
+[instructions](https://cloud.google.com/alloydb/docs/database-create). Note the database
+name. 
+
+1. Create a user in your database by following these 
+[instructions](https://cloud.google.com/alloydb/docs/database-users/about). Note the username. 
+
+1. Create a [service account](https://cloud.google.com/iam/docs/understanding-service-accounts) with the 'AlloyDB Client' permissions.
+
+
+1. Use the information noted in the previous steps:
```

Comment:
Should all these steps be `1.`? Seems to render with proper numbers.

Reply:
in markdown, for an ordered list, if you just put `1.`, it will add the correct numbers for you. This makes it easier to edit if you want to insert a step between two other steps.

Reply:
TIL 😃